src/Voxel.MiddyNet.Tracing.ApiGatewayMiddleware/ApiGatewayHttpApiV2TracingMiddleware.cs

```diff
@@ -6,6 +6,7 @@ namespace Voxel.MiddyNet.Tracing.ApiGatewayMiddleware
 {
     public class ApiGatewayHttpApiV2TracingMiddleware : ILambdaMiddleware<APIGatewayHttpApiV2ProxyRequest, APIGatewayHttpApiV2ProxyResponse>
     {
+        private const string TraceContextKey = "TraceContext";
```

Comment:
Maybe we can make this public and avoid magic strings?

Reply:
I haven't found a way to use a public const of this middleware from the lambda using it.
Instead, I have created a public const in the tracecontext class.

Reply:
Resolved by vgaltes